conans/tools.py

```diff
@@ -146,7 +164,7 @@ def vcvars_command(settings):
     param = "x86" if settings.arch == "x86" else "amd64"
     existing_version = os.environ.get("VisualStudioVersion")
     if existing_version:
-        command = ""
+        command = "echo Conan:vcvars already set"
```

Comment:
Could you add a test that checks that output to ensure that it's running ok when vcvars is already set? I know that probably it's already tested, but just for ensure it.